examples/azureml/dockerfile

```diff
@@ -23,9 +23,7 @@ RUN pip install --no-cache-dir --use-feature=2020-resolver \
     pandas \
     tqdm \
     tensorboard \
-    azureml-sdk \
+    azureml-core \
```

Comment:
didn't even realize but you could just use pymarlin[plugins] as an azureml pip/conda dependency instead of building from dockerfile but it's all the same at the end of the day.